src/monitor-and-troubleshoot/log-streaming/configure-collector.md

```diff
@@ -29,7 +29,13 @@ To receive logs in Datadog or Splunk, you must set up an OpenTelemetry Collector
 
 1.  Run the OpenTelemetry Collector. 
 
-**Note**: If the OpenTelemetry Collector implementation is OLTP over HTTP, according to the [OpenTelemetry specification](https://opentelemetry.io/docs/specs/otlp/#otlphttp-request), the default URL path for requests that carry log data is ``v1/logs``. If using OTLP/HTTP, you must add this path to the URL endpoint for a successful connection and data transfer.
+<div class="info" markdown="1">
+
+* If the OpenTelemetry Collector implementation is OLTP over HTTP, according to the [OpenTelemetry specification](https://opentelemetry.io/docs/specs/otlp/#otlphttp-request) then, the default URL path for requests that carry log data is ``v1/logs``. If you are using OTLP/HTTP, you must add this path to the URL endpoint for a successful connection and data transfer.
+
+* While monitoring to avoid unexpected connection failure, you must keep track and renew the SSL/TLS certificate used in the OpenTelemetry collector before the expiration date.
```

Comment:
```suggestion
* To avoid unexpected connection failures, you should keep track and renew the SSL/TLS certificate used in the OpenTelemetry collector before the expiration date.
```